docs/source/conf.py

```diff
@@ -98,7 +97,6 @@
 
 # Show docstring for special members
 autodoc_default_options = {
-    "special-members": True,
```

Comment:
was there a specific reason to move this to the individual classes? how did you check that no class was forgotten that currently documents special members?

Reply:
yeah for some reason we're getting all special methods defined in `object` like `__dir__` too. 

> how did you check that no class was forgotten that currently documents special members?

I just relied on build warnings, which would happen if a reference wasn't found. So I could've definitely missed a few

Reply:
Ok I found a better fix